javascript-functions/main.js

```diff
@@ -0,0 +1,43 @@
+function convertMinutesToSeconds(minutes) {
+  var seconds = minutes * 60;
+  return minutes +' minutes is ' + seconds + ' seconds.' ;
+}
+
+var convertMinutesToSecondsResults = convertMinutesToSeconds(3);
+console.log('converMinutesToSecondsResults is: ', convertMinutesToSecondsResults);
+
+function greet(name) {
+  var greeting = 'Hey ' + name;
+  return greeting;
+}
+
+var greetResults = greet('Travis');
+console.log('greetResults is:', greetResults);
+
+function getArea(width, height) {
+  var area = width * height;
+  return 'The total area is: ' + area;
+}
+
+var getAreaResults = getArea(6, 3);
+console.log('getAreaResults is: ', getAreaResults);
+
+function getFirstName(person) {
+  var person = {
```

Comment:
Instead of defining the object here, you should be defining it when you call your function, because you're passing the object as a parameter